eb_sqs/worker/service.py

```diff
@@ -79,6 +88,9 @@ def process_messages(self, queues, worker, static_queues):
         # type: (list, Worker, list) -> None
 
         for queue in queues:
+            if self._exit_gracefully:
```

Comment:
Should we consider moving this into `for msg in messages` and checking it between messages? That way it will be more stable when processing bigger batches.

Reply:
Currently we can't, because we delete the whole 10 message batch at the end of processing.

I guess it's possible to only delete the ones we processed, or re-queue the unprocessed ones, but it's a bigger refactoring.